src/integrationtests/resources/com/aws/greengrass/integrationtests/logmanager/randomlyNamedFilesWithoutComponentsConfig.yaml

```diff
@@ -0,0 +1,20 @@
+---
+services:
+  aws.greengrass.LogManager:
+    configuration:
+      periodicUploadIntervalSec: 10
+      logsUploaderConfiguration:
+        componentLogsConfiguration:
+          - componentName: 'UserComponentB'
+            logFileRegex: '\w*UserComponentB\w*.\w*'
+            logFileDirectoryPath: '{{logFileDirectoryPath}}'
+            minimumLogLevel: 'INFO'
+            diskSpaceLimit: '25'
+            diskSpaceLimitUnit: 'MB'
+            deleteLogFileAfterCloudUpload: 'true'
```

Comment:
don't delete after upload. I want to ensure that the log files are not uploading duplicates, even if their name changes (after being uploaded)